docker-compose.yml

```diff
@@ -16,8 +16,7 @@ services:
     container_name: enterprise_subsidy.memcache
 
   app:
-    # Uncomment this line to use the official enterprise_subsidy base image
-    # image: openedx/enterprise_subsidy
+    image: openedx/enterprise-subsidy
```

Comment:
Makes it so we're not maintaining two differently-named docker images for subsidy.